docs/decisions/0013-library-reference-content-block.rst

```diff
@@ -0,0 +1,61 @@
+Referencing Content Blocks in Library V2
+--------------------------------------------------
+
+Status
+=======
+Pending
+
+Context
+=======
+Currently, the courses are being stored in the Modulestore and the libraries are stored in both Modulestore and the Blockstore. We want the library content to be stored in the Blockstore.
```

Comment:
- Usage of `Modulestore` or `modulestore` should be consistent throughout the ADR.
- It'd be worth calling out that the libraries stored in the modulestore are the legacy v1 content libraries, while the blockstore-based content libraries are the v2 libraries.
- `We want the library content to be stored in the Blockstore. ` -> `All the library content should instead be stored in the blockstore.` In other words, `we want` should be avoided altogether this ADR. The ADR's purpose is to document what `we want`, so no need to state it.

Reply:
Done